hillel-expense-tracker-repo/src/test/java/ua/ithillel/expensetracker/repo/exception/ExpenseTrackerPersistingExceptionTest.java

```diff
@@ -0,0 +1,31 @@
+package ua.ithillel.expensetracker.repo.exception;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+class ExpenseTrackerPersistingExceptionTest {
```

Comment:
no need to create this test class, since the exception class doesn't provide any functionalities